bitnami/mariadb/10.9/debian-11/rootfs/opt/bitnami/scripts/libmariadb.sh

```diff
@@ -515,6 +518,7 @@ mysql_execute_print_output() {
     local -a args=()
     if [[ -f "$DB_CONF_FILE" ]]; then
         args+=("--defaults-file=${DB_CONF_FILE}")
+        [[ -f "$DB_EXTRA_CONF_FILE" ]] && args+=("--defaults-extra-file=${DB_EXTRA_CONF_FILE}")
     fi
```

Comment:
Does `--defaults-extra-file` require `--defaults-file`? If not, I would suggest:
```suggestion
    [[ -f "$DB_CONF_FILE" ]] && args+=("--defaults-file=${DB_CONF_FILE}")
    [[ -f "$DB_EXTRA_CONF_FILE" ]] && args+=("--defaults-extra-file=${DB_EXTRA_CONF_FILE}")
```

Reply:
Anywhere else in the code, there seems to be no decision making as of the `--defaults-file` (it's kinda expected to exist / be set). I wouldn't recommend introducing it in this single place, or - if introduced - should be done on all occurrences (separate MR).